frontend-friends/src/components/list/List.js

```diff
@@ -0,0 +1,37 @@
+import "./list.scss";
+import data from "./dummy-data.json";
+import { useState } from "react";
+
+export default function List() {
+
+    const [cases] = useState(data);
+
+    return (
+        <table>
+            <thead>
+                <tr className="list-header">
+                    <th>
+                        ÄRENDENUMMER
+                    </th>
+                    <th>
+                        ÄRENDE
+                    </th>
+                    <th>
+                        TID
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                {cases.map((currCase)=> 
+                <tr>
```

Comment:
Tror det går att fixa kodlukten med att lägga in key={currCase.id} på <tr>.

Reply:
+1